dict/prh-idiomatic-usage.yml

```
@@ -994,6 +1014,26 @@ rules:
         to: カスタムマスター
       - from: カスタムマスター
         to: カスタムマスター
+  - expected: 基本機能
+    pattern:
+      - /本体アプリ|本体/
```

「本体」だけだと誤検知出そうなので
このようにするとどうでしょう
↓サンプル
https://regex101.com/r/XFkyZi/1

前半は「本体アプリ」の直前に「SmartHR」が0または1個以上登場する場合に「本体アプリ」にマッチ
後半は「本体」の直前に「SmartHR」が1個登場し、かつ「本体」の直後に「アプリ」が0または1個以上登場する場合にマッチ

```suggestion
      /((?<=SmartHR)?本体アプリ)|((?<=SmartHR)本体(?=アプリ)?)/
```

---

いや、ちょっとおかしいので修正します

---

:kanisugi: ありがとうございます！